cypress/integration/collections_spec.js

```diff
@@ -0,0 +1,106 @@
+import { shouldBeRedirectedToLogin } from '../support/assertions';
+
+describe('Dashboard Collections Page', () => {
+  describe('When not logged in', () => {
+    it('should redirect to login page', () => {
+      cy.visit('/#/collections');
+      shouldBeRedirectedToLogin();
+    });
+  });
+
+  describe('When logged in', () => {
+    beforeEach(() => {
+      cy.login();
+      cy.task('resetState');
+    });
+
+    after(() => {
+      cy.task('resetState');
+    });
+
+    it('displays a link to view collections', () => {
+      cy.visit('/');
+
+      cy.contains('nav li a', 'Collections').should('exist').as('collections');
+      cy.get('@collections').should('have.attr', 'href', '#/collections');
+      cy.get('@collections').click();
+
+      cy.url().should('include', 'collections');
+      cy.get('.heading--xlarge').should('have.text', 'Collections');
+
+      cy.get('table tbody tr').its('length').should('be.eq', 5);
+    });
+
+    it('collections page displays a button to add a new collection', () => {
+      const name = 'TESTCOLLECTION';
+      const version = '006';
+      cy.visit('/#/collections');
+
+      cy.get('.heading--large').should('have.text', 'Collection Overview');
+      cy.contains('a', 'Add a Collection').should('exist').as('addCollection');
+      cy.get('@addCollection').should('have.attr', 'href', '#/collections/add');
+      cy.get('@addCollection').click();
+
+      // fill the form and submit
+      const collection = '{{}"name":"TESTCOLLECTION","version":"006","dataType":"TESTCOLLECTION", "duplicateHandling":"error"}';
+      cy.get('textarea').type(collection, {force: true});
+      cy.get('form').get('input').contains('Submit').click();
+
+      // displays the new collection
+      cy.get('.heading--xlarge').should('have.text', 'Collections');
+      cy.get('.heading--large').should('have.text', `${name} / ${version}`);
+      cy.url().should('include', `#/collections/collection/${name}/${version}`);
```

Comment:
Do we want to add assertions for the correct `duplicateHandling` or any of the other specific properties?